src/client/gl-base-technique.cc

```diff
@@ -79,6 +80,23 @@ GlBaseTechnique::Commit()
     update_rendering_queue_->Push(std::move(command));
   }
 
+  if (!pending_.gl_textures.empty()) {
```

Comment:
We have to commit associated textures before hand. See the beginning of GlBaseTechnique::Commit() for the example of other resources (program, vertex_array).